src/library/Uno.Material/Styles/Controls/v2/TextBlock.xaml

```diff
@@ -178,7 +178,7 @@
         <Setter Property="FontSize"
                 Value="11" />
         <Setter Property="LineHeight"
-                Value="6" />
+                Value="16" />
```

Comment:
Ah there's a typo in the M3 guidelines, that's why we had it at 6

![image](https://user-images.githubusercontent.com/4793020/169633183-b41e04ee-0e93-4cde-826c-ba16992b1dfd.png)